assets/src/admin/site-scan-notice/notice.js

```diff
@@ -71,13 +71,15 @@ export function SiteScanNotice() {
 		let elements = [
 			pluginsWithAmpIncompatibility.length > 0 ? (
 				<PluginsWithAmpIncompatibility
+					key={pluginsWithAmpIncompatibility.length}
```

Comment:
```suggestion
					key="plugins"
```

Reply:
Or should it rather be:

```suggestion
					key={`plugins${pluginsWithAmpIncompatibility.length}`}
```